modules/xz/5.4.5/presubmit.yml

```diff
@@ -0,0 +1,30 @@
+matrix: &matrix
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - windows
+  bazel: [6.x, 7.x, rolling]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      # - "@xz//:xz"
+      - "@xz//:lzma"
+      # - "@xz//:xzdec"
+      # - "@xz//:lzmadec"
+bcr_test_module:
+  module_path: ""
+  matrix: *matrix
+  tasks:
+    run_test_module:
+      name: Run test module
+      platform: ${{ platform }}
+      bazel: ${{ bazel }}
+      test_targets:
+        - all_tests
```

Comment:
@Attempt3035 What does `all_tests` do? Would propse this presubmit file:
```
matrix:
  platform:
    - debian10
    - debian11
    - macos
    - macos_arm64
    - ubuntu2004
    - ubuntu2204
    - windows
  bazel: [6.x, 7.x, rolling]
tasks:
  verify_targets:
    name: Verify build targets
    platform: ${{ platform }}
    bazel: ${{ bazel }}
    build_targets:
      - "@xz//:lzma"
```

Reply:
all_tests is the test group target that runs all the cc_library tests

Reply:
Just having as you proposed would only verify build but not functionality via doing unit tests I believe?

Reply:
@Attempt3035  ah okay... then ignore my comment ;)